framework/src/main/java/org/apache/drill/test/framework/ColumnList.java

```diff
@@ -88,8 +93,36 @@ public int hashCode() {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     for (int i = 0; i < values.size() - 1; i++) {
+      int type = (Integer) (types.get(i));
+      if (categories.contains("smoke_simba_jdbc") && (type == Types.VARCHAR)) {
+        String s1 = String.valueOf(values.get(i));
+        // if the field has a JSON string or list, then remove newlines so that
+        // each record fits on a single line in the actual output file.
+        // sometimes Drill returns records in different orders, and by
+        // ensuring each reoord is on a single line, they can be matched in any
+        // order
+        if ((s1.length() > 0) &&
+           ((s1.charAt(0) == '{') || (s1.charAt(0) == '[')) ) {
+          s1 = s1.replace("\n","");
```

Comment:
Have you considered replacing "\n" (new line) with " " (space) instead of "" (empty string)? I'm concerned some strings with new lines will end up concatenated instead of simple eliminating new lines. 
For ex: 
"How are
you" 
should not end up as "How areyou", but instead as "How are you".